Cargo.toml

```diff
@@ -35,6 +35,7 @@ zip = "0.6"
 
 [features]
 closure = []
+embed = []
```

Comment:
Just a reminder to put in the doc that this feature will only work on linux OS for now

Reply:
Done